src/containers/Home.jsx

```diff
@@ -1,21 +1,79 @@
-import React, {Component} from "react";
+/*
+Home page for SE Society
+Last Edited: Tony Zhao 2020-04-22
+*/
+
+import React, {Component, Fragment} from "react";
+import {Button, Container, Jumbotron, Row, Col} from "react-bootstrap";
+import EventList from "../components/EventList/EventList";
```

Comment:
`/EventList/EventList` is silly
move ur sass 
or move your eventlist to a file named `index.js` and you will be able to export it from the `/EventList` folder directly

Reply:
The definitions inside `EventList.scss` were all overridden so I moved `EventList/EventList.jsx` to `EventList.jsx`